tests/acceptance/repo/settings-test.js

```diff
@@ -26,13 +26,16 @@ module('Acceptance | repo settings', function (hooks) {
       login: 'org-login',
     });
 
+    this.server.create('allowance', {subscription_type: 1});
+    this.server.create('allowance', {subscription_type: 1});
```

Comment:
Is it a duplication? If it's intended, you could use
```suggestion
    this.server.createList('allowance', 2, {subscription_type: 1});
```
Does it make sense?

Reply:
It's not a duplication. Here I need 2 records of allowance. Thanks for hint level pro! :-)